db/migrate/20160615205001_add_paid_at_to_auctions.rb

```diff
@@ -0,0 +1,24 @@
+class AddPaidAtToAuctions < ActiveRecord::Migration
+  def up
+    add_column :auctions, :paid_at, :datetime
+
+    execute <<-SQL
+      UPDATE auctions
+      SET paid_at = CURRENT_TIMESTAMP
```

Comment:
yeah good call I thought of that...do we have a way of knowing when past auctions were paid? Guessing that process would be manual? @adelevie